src/XPlot.Plotly.Interactive/NewPlotlyChartCommand .fs

```diff
@@ -0,0 +1,31 @@
+﻿namespace XPlot.Plotly.Interactive.PowerShell.Commands
+
+open XPlot.Plotly
+open System.Management.Automation
+
+[<Cmdlet(VerbsCommon.New, "PlotlyChart")>]
+[<OutputType("XPlot.Plotly.Chart")>]
+[<Alias("npc")>]
+type NewPlotlyChartCommand() =
+    inherit PSCmdlet()
+
+    [<Parameter(Mandatory = true, Position = 0, ValueFromPipeline = true)>]
+    member val Trace:Trace[] = null with get, set
+
+    [<Parameter(Position = 1)>]
+    member val Title = "" with get, set
+
+    [<Parameter(Position = 1)>]
+    member val Layout = Unchecked.defaultof<Layout.Layout> with get, set
+
+    override _.BeginProcessing() = ()
+
+    override _.ProcessRecord() = ()
```

Comment:
Please do the same as the original implementation. This makes sure the piping behavior is as expected.
```
        private List<Trace> _traces;

        /// <summary>
        /// BeginProcessing override.
        /// </summary>
        protected override void BeginProcessing()
        {
            _traces = new List<Trace>();
        }
        
        /// <summary>
        /// ProcessRecord override.
        /// </summary>
        protected override void ProcessRecord()
        {
            _traces.AddRange(Trace);
        }
```